osquery/tests/integration/tables/process_open_files.cpp

```diff
@@ -12,29 +11,53 @@
 // Sanity check integration test for process_open_files
 // Spec file: specs/posix/process_open_files.table
 
+#include <unistd.h>
```

Comment:
Why do you need `unistd.h` include there?

Reply:
getpid()